packages/commonwealth/client/scripts/views/pages/view_proposal/proposal_components.tsx

```diff
@@ -18,25 +13,13 @@ type ProposalSubheaderProps = {
 
 export const ProposalSubheader = (props: ProposalSubheaderProps) => {
   const { proposal } = props;
-  const [linkedThreads, setLinkedThreads] =
-    // @ts-expect-error <StrictNullChecks/>
-    useState<{ id: number; title: string }[]>(null);
-
-  useNecessaryEffect(() => {
-    if (!linkedThreads) {
-      axios
-        .post(`${SERVER_URL}/linking/getLinks`, {
-          link: {
-            source: LinkSource.Proposal,
-            identifier: proposal.identifier,
-          },
-          jwt: userStore.getState().jwt,
-        })
-        .then((response) => {
-          setLinkedThreads(response.data.result.threads);
-        });
-    }
-  }, []);
+  // const { data: linkedThreads } = useGetThreadsByLinkQuery({
+  //   link: {
+  //     source: LinkSource.Proposal,
+  //     identifier: proposal.identifier,
+  //   },
+  //   enabled: !!proposal.identifier,
+  // });
```

Comment:
what happens here? Do not commit commented code

Reply:
Looks like we are not using this hook at all